services/core-web/src/components/dashboard/incidentsHomePage/IncidentsHomePage.js

```diff
@@ -162,6 +167,14 @@ export class IncidentsHomePage extends Component {
     });
   };
 
+  handleDeleteMineIncident = (values) => {
+    this.props.deleteMineIncident(values.mine_guid, values.mine_incident_guid).then(() => {
+      this.props.fetchIncidents(this.state.params).then(() => {
+        this.setState({ incidentsLoaded: true });
```

Comment:
I think you'll want to set this to `false` right before `this.props.fetchIncidents`. That way the "loading effect" will be achieved.